tanaguru-engine/tanaguru-api/src/main/resources/db/changelogs/20210105_changelog_updateTestHierarchy_engineVersion.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd"
+        objectQuotingStrategy="QUOTE_ONLY_RESERVED_WORDS">
+        
+    <changeSet id="update_testhierarchy_engineversion" author="lpedrau">
+        <addColumn tableName="test_hierarchy">
+            <column name="webext_engine_id" type="bigint"/>
+        </addColumn>
+        <addForeignKeyConstraint
+                baseTableName="test_hierarchy"
+                baseColumnNames="webext_engine_id"
+                constraintName="fk_test_hierarchy_engine_version_webext_engine"
+                referencedTableName="webext_engine"
+                referencedColumnNames="id"/>
+                           
+        <rollback>
+            <dropColumn tableName="test_hierarchy" columnName="webext_engine_id"/>
+        </rollback>
+    </changeSet>
```

Comment:
Tu peux mettre plusieurs changeset dans un meme changelog, essaies de prendre cette habitude quand ils font partie de la meme fonctionnalité

Reply:
d'accord c'est noté